lighthouse-core/audits/installable-manifest.js

```diff
@@ -5,38 +5,130 @@
+  'no-icon-available': `Downloaded icon was empty or corrupted`,
+  /** Error message explaining that the specified application platform is not supported on Android. */
+  // eslint-disable-next-line max-len
+  'platform-not-supported-on-android': `The specified application platform is not supported on Android`,
+  /** Error message explaining that a Play store ID was not provided. */
+  'no-id-specified': `No Play store ID provided`,
+  /** Error message explaining that the Play Store app URL and Play Store ID do not match. */
+  'ids-do-not-match': `The Play Store app URL and Play Store ID do not match`,
+  /** Error message explaining that the app is already installed. */
+  'already-installed': `The app is already installed`,
+  /** Error message explaining that a URL in the manifest contains a username, password, or port. */
+  'url-not-supported-for-webapk': `A URL in the manifest contains a username, password, or port`,
+  /** Error message explaining that the page is loaded in an incognito window. */
+  'in-incognito': `Page is loaded in an incognito window`,
+  // TODO: perhaps edit this message to make it more actionable for LH users
+  /** Error message explaining that the page does not work offline. */
+  'not-offline-capable': `Page does not work offline`,
+  /** Error message explaining that service worker could not be checked without a start_url. */
+  // eslint-disable-next-line max-len
+  'no-url-for-service-worker': `Could not check service worker without a 'start_url' field in the manifest`,
+  /** Error message explaining that the manifest specifies prefer_related_applications: true. */
+  'prefer-related-applications': `Manifest specifies prefer_related_applications: true`,
+  /** Error message explaining that prefer_related_applications is only supported on Chrome Beta and Stable channe
+               on Android. */
+  // eslint-disable-next-line max-len
+  'prefer-related-applications-only-beta-stable': `prefer_related_applications is only supported on Chrome Beta and Stable channe 
+                on Android.`,
+  /** Error message explaining that the manifest contains 'display_override' field, and the first supported display
+               mode must be one of 'standalone', 'fulcreen', or 'minimal-ui. */
+  // eslint-disable-next-line max-len
+  'manifest-display-override-not-supported': `Manifest contains 'display_override' field, and the first supported display 
+                mode must be one of 'standalone', 'fulcreen', or 'minimal-ui`,
+  /** Error message explaining that the web manifest's URL changed while the manifest was being downloaded by the browser. */
+  'manifest-location-changed': `Manifest URL changed while the manifest was being fetched.`,
+  /** Warning message explaining that the page does not work offline. */
+  'warn-not-offline-capable': `Page does not work offline. The page will not be regarded
+              as installable after Chrome 93, stable release August 2021.`,
```

Comment:
annoying thing i just noticed... we have spaces in these strings. keeping them probably wouldn't mess with the end UI, but it'd likely confuse our translators. :)

a few options..

1) there's this awkward pattern:

```js
  description: ' ..................................... ' + 
    '..........';
```

but it's terrible.

2) you have a few `  // eslint-disable-next-line max-len` in there but it might be annoying to sprinkle them everywhere

3) I think it'd be fine to do

```
  /* eslint-disable max-len */
.....the big UI strings section........
  /* eslint-enable max-len */
```

Reply:
I like the 3rd :)